src/runtime/api-channel.ts

```diff
@@ -203,17 +203,35 @@ class ThingMapper {
   }
 }
 
+export abstract class Port {
```

Comment:
I'd rather keep the abstraction for Java port, because it is implemented with Pipe shell specific deviceClient and shellApi channels, but i switched back to just using MessagePort for the JS PEC, WDYT?

Reply:
I think you should use MessagePort everywhere, it's not hard to adapt Port for the Java case.

Reply:
oh, i think i finally understood what you meant all the way - got rid of the Port class, using MessagePort everywhere.